src/routes/home/Home.js

```diff
@@ -648,9 +652,12 @@ class Home extends React.Component {
       this.attachmentPlates.set(attachmentFile, result);
       return result;
     } catch (err) {
+      if (err == 'license plate (image was read, but no plates were found)') {
+        throw err;
+      }
       console.error(err.stack);
 
-      throw 'license plate'; // eslint-disable-line no-throw-literal
+      throw 'license plate (image could not be read)'; // eslint-disable-line no-throw-literal
```

Comment:
I'm not sure if using parenthetical text is the best way to convey what happened, especially if GPS and/or timestamp also could not be read. For one, users may not read all the text, so it may not be clear what happened with the plate-reading.